regulations/templates/regulations/partials/sidebar_left/base.html

```diff
@@ -0,0 +1,40 @@
+{% load static %}
+
+{% block controls %}
+    <div class="toc-controls">
+        <button data-set-state="collapsed" data-state-name="left-sidebar">
+            <img src="{% static 'images/sidebar-collapse.svg' %}" />
+            <span>Collapse</span>
+        </button>
+        <button data-set-state="expanded" data-state-name="left-sidebar">
+            <img src="{% static 'images/sidebar-expand.svg' %}" />
+        </button>
+    </div>
+{% endblock %}
+
+{% block jump-to %}
+    <div class="jump-to">
+        <div class="jump-to-label">Jump to Regulation Section</div>
+
+        <form method="GET" action="{% url 'goto' %}">
+        <input name="{{ reg_part }}-version" type="hidden" required value={{ version }}>
+        <input name="title" type="hidden" required value={{ title }}>
```

Comment:
This is coming up blank in this case breaking to the Jump To behavior. We may have to rethink how this works here since Search doesn't have a specific title to fall back on.

Reply:
jump-to removed from search sidebar